docker/Dockerfile.alpine

```
@@ -1,4 +1,5 @@
-FROM alpine:3.16 as builder
+ARG ALPINE_VERSION=alpine:3.16
```

```suggestion
ARG ALPINE_VERSION=alpine:3.20
```

As 3.16 no longer supported https://alpinelinux.org/releases/